### ISSUE TITLE
Web beacons not detected

### ISSUE THREAD

**Issue by bfg1981** [andersju/webbkoll#17]
It would be nice if web beacons like Facebook Pixel etc. were also detected.

**Comment by andersju** [comment]
Agreed. Right now one can deduce that from third-party requests / cookies (looks like Facebook Pixel triggers a request to https://connect.facebook.net/en_US/fbevents.js and sets a cookie called _fbp) - third-party requests get a classification thanks to https://github.com/disconnectme/disconnect-tracking-protection - but it would be great to present what this really means ("this site uses Google Analytics/Facebook Pixel/...").

We used to have something like this for a few services, but I'd prefer not to have us maintain our own list if possible. I wonder if there's a good, open list somewhere? (Like, something more detailed than Disconnect's list, for at least the most common beacons/trackers)
